src/compiler/types.ts

```diff
@@ -1807,8 +1807,8 @@ namespace ts {
 
     export interface CatchClause extends Node {
         kind: SyntaxKind.CatchClause;
-        parent?: TryStatement;
-        variableDeclaration: VariableDeclaration;
+        parent?: TryStatement; // We make this optional to parse missing try statements

```

Comment:
The comment is unnecessary. We only re-introduce `parent` here to refine its type for methods in the checker. `parent` is optional because it parent pointers are not always set, especially during tree transformations.